src/Symfony/Component/VarDumper/Cloner/Data.php

```diff
@@ -367,6 +367,14 @@ private function dumpItem(DumperInterface $dumper, Cursor $cursor, array &$refs,
                         $cut += \count($children);
                     }
                     $cursor->skipChildren = false;
+                    // Add uninitialized properties
+                    foreach ($item->attr["uninitialized"] as $uninitializedProperty) {
```

Comment:
shouldn't this be done inside `dumpChildren` instead ? Otherwise, things will be weird when the cursor is meant to dump without children.